mappings/net/minecraft/world/biome/source/BiomeAccessor.mapping

```diff
@@ -0,0 +1,16 @@
+CLASS net/minecraft/class_4543 net/minecraft/world/biome/source/BiomeAccessor
```

Comment:
Thoughts on `BiomeReader` as an alternative? (random thought, not sure about it)

Reply:
This is like reading from the biome array, not really from the world